presto-product-tests/src/main/java/com/facebook/presto/tests/hive/TestSqlStandardAccessControlChecks.java

```diff
@@ -0,0 +1,175 @@
+
+        assertThat(() -> queryExecutorForBob.executeQuery(format("ALTER TABLE %s ADD COLUMN year bigint", tableName))).
+                failsWithMessage(format("Access Denied: Cannot add a column to table default.%s", tableName));
+        queryExecutorForAlice.executeQuery(format("ALTER TABLE %s ADD COLUMN year bigint", tableName));
+    }
+
+    @Test(groups = {AUTHORIZATION, HIVE_CONNECTOR, PROFILE_SPECIFIC_TESTS})
+    public void testAccessControlCreateView()
+    {
+        String tableName = "alice_owned_table";
+        String viewName = "bob_view";
+        QueryExecutor queryExecutorForAlice = connectToPresto("alice@presto");
+        QueryExecutor queryExecutorForBob = connectToPresto("bob@presto");
+
+        queryExecutorForAlice.executeQuery(format("DROP TABLE IF EXISTS %s", tableName));
+        queryExecutorForAlice.executeQuery(format("CREATE TABLE %s(month bigint, day bigint)", tableName));
+
+        queryExecutorForBob.executeQuery(format("DROP VIEW IF EXISTS %s", viewName));
+        assertThat(() -> queryExecutorForBob.executeQuery(format("CREATE VIEW %s AS select * from %s", viewName, tableName))).
+                failsWithMessage(format("Access Denied: Cannot select from table default.%s", tableName));
+        queryExecutorForAlice.executeQuery(format("GRANT SELECT ON %s TO bob", tableName));
+        assertThat(() -> queryExecutorForBob.executeQuery(format("CREATE VIEW %s AS select * from %s", viewName, tableName))).
+                failsWithMessage(format("Access Denied: Cannot create view that selects from %s", tableName));
+
+        queryExecutorForAlice.executeQuery(format("REVOKE SELECT ON %s FROM bob", tableName));
+        queryExecutorForAlice.executeQuery(format("GRANT SELECT ON %s TO bob WITH GRANT OPTION", tableName));
+        executeWith(createViewAs(viewName, "SELECT * FROM alice_owned_table", queryExecutorForBob), view -> {
+            assertThat(queryExecutorForBob.executeQuery(format("SELECT * FROM %s", view.getName()))).
+                    hasNoRows();
+        });
+    }
+
+    @Test(groups = {AUTHORIZATION, HIVE_CONNECTOR, PROFILE_SPECIFIC_TESTS})
+    public void testAccessControlDropView()
+    {
+        String tableName = "alice_owned_table";
+        String viewName = "alice_view_for_drop";
+        QueryExecutor queryExecutorForAlice = connectToPresto("alice@presto");
+        QueryExecutor queryExecutorForBob = connectToPresto("bob@presto");
+
+        queryExecutorForAlice.executeQuery(format("DROP TABLE IF EXISTS %s", tableName));
+        queryExecutorForAlice.executeQuery(format("CREATE TABLE %s(month bigint, day bigint)", tableName));
+        queryExecutorForAlice.executeQuery(format("CREATE VIEW %s AS SELECT * FROM %s", viewName, tableName));
+
+        assertThat(() -> queryExecutorForBob.executeQuery(format("DROP VIEW %s", viewName))).
+                failsWithMessage(format("Access Denied: Cannot drop view default.%s", viewName));
```

Comment:
Nope, there is no such `DROP VIEW` privilege that one can grant. Only owners of the view can drop the view.